src/palace/manager/celery/tasks/playtime_entries.py

```diff
@@ -0,0 +1,379 @@
+import csv
+import os
+import tempfile
+import uuid
+from collections import defaultdict
+from collections.abc import Iterable
+from datetime import date, datetime, timedelta
+from io import TextIOWrapper
+from typing import Any, Protocol
+
+import pytz
+from celery import shared_task
+from sqlalchemy import and_, distinct, false, select, true
+from sqlalchemy.orm import Query, Session
+from sqlalchemy.sql.functions import coalesce, count, max as sql_max, sum
+
+from palace.manager.api.config import Configuration
+from palace.manager.celery.task import Task
+from palace.manager.service.celery.celery import QueueNames
+from palace.manager.service.google_drive.google_drive import GoogleDriveService
+from palace.manager.sqlalchemy.model.collection import Collection
+from palace.manager.sqlalchemy.model.identifier import Identifier
+from palace.manager.sqlalchemy.model.library import Library
+from palace.manager.sqlalchemy.model.time_tracking import PlaytimeEntry, PlaytimeSummary
+from palace.manager.util.datetime_helpers import previous_months, utc_now
+from palace.manager.util.uuid import uuid_encode
+
+
+@shared_task(queue=QueueNames.high, bind=True)
```

Comment:
Do we want to the tasks to execute ASAP?  Not sure. Thoughts?

Reply:
I'd like to reserve the high queue for tasks that a user is actively waiting for. I think this should go into `default`.